model.py

```diff
@@ -328,13 +328,18 @@ class SessionManager(object):
     engine_for_url = {}
 
     @classmethod
-    def engine(cls, url=None):
+    def engine(cls, url=None, session=None):
```

Comment:
I doesn't look like session is used here.

Reply:
Thanks - 'twas a throwback.